### DISCUSSION TITLE
격자? 그리드? 뷰에 page가 적용되었을때 말하는 사람이 보였으면 좋겠어요

### DISCUSSION THREAD

**Discussion by act-lamos** [metamong-friends/metamong-friends.github.io#62] [제안하기]
사용자 카메라만 나오는 화면에서 여러명이 화상회의 할 때 페이징이 적용되는데

말하고 있는 사람이 다음 페이지에 있는 경우 직접 페이지 이동을 해야해서 불편합니다.

**Comment by prism27** [comment]
안녕하세요!  TJ 입니다^^.

좋은 제안 감사드립니다.

그리드뷰 화면에서 화자의 노출이 되지 않는 케이스에 대한 불편함이 있는 상황을 잘 인지하였습니다.

음.. 저희가 소리 유무만을 체크해서 유저 순서를 변경하다 보면 
너무 잦은 변경이나, 동작에 혼선이 오는 케이스들이 생길 것 같긴 하네요. 

마이크를 on 한 유저들  우선 최대한 노출이 되도록 초기로드, 업데이트해 주는 것이 하나의 대안이 될 수 있을 것 같습니다.
이 경우 모두가 키고 회의하는 경우는 커버되지 않으니... 아니면 page 당 유저 수를 좀 더 늘릴 수 있도록 하는 안도 있을 수 있겠습니다.

여러 사용 케이스를 고려해 개선 방안을 고민해 보도록 하겠습니다.

감사합니다!
